raiden-network/light-client
# Feature/dapp prevent to load version directly from server

**Thank you for submitting this pull request :)**

Fixes #3024

**Short description**
Checkout the issue and the commit messages for details.

**Definition of Done**


**Steps to manually test the change (dApp)**

1. Build productive version of the dApp for current version (`yarn build`)
2. Service build locally (e.g. `npx serve ./dist`)
3. Ensure the browser has no data on the now served location (in Chrome use dev-tools application, `Clear Site Data`)
4. Load current version of the local dApp and wait for it to install
5. Adapt the version to a higher number in `package.json` (e.g. `3.0.0`)
6. Rebuild the dApp (`yarn build`)
7. Force-reload the dApp in the browser (`Shift` + `F5` in Chrome)
8. Ensure that the dApp became temporally blocked (looks a bit squishy)
9. Ensure that the dApp has reloaded itself again
10. Ensure the dApp now shows a version update and the loaded version is the "old" one (at the bottom of the account screen)
